### DISCUSSION TITLE
Allow changing default "master" branch name in Insomnia Sync

### DISCUSSION THREAD

**Discussion by d19dotca** [Kong/insomnia#6503] [Ideas]
Filing on behalf of a customer who would like to be able to rename the "master" branch which is created by default for the Insomnia Sync feature to "main" instead. Ideally this would be open to having the ability to rename it anything they'd like.

**Comment by JC5** [comment]
Yes please. Having a branch named `master` in 2023 (2025!) feels very 1990s.
